meta-rdk-mtk-bpir4/recipes-ccsp/rdk-wifi-libhostap/rdk-wifi-libhostap.bbappend

```diff
@@ -3,20 +3,89 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 LIC_FILES_CHKSUM_remove = "file://source/hostap-2.10/README;md5=e3d2f6c2948991e37c1ca4960de84747"
 LIC_FILES_CHKSUM = "file://source/hostap-2.11/README;md5=6e4b25e7d74bfc44a32ba37bdf5210a6"
 
+DEPENDS_append = " ucode"
+
+SRC_URI = "git://w1.fi/hostap.git;protocol=https;branch=main;destsuffix=${S}/source/hostap-${HOSTAPD_PV};name=${HOSTAPD_PV}"
+
```

Comment:
It is resetting libhostap to a new source location and new version
SRCREV_2.11 = "96e48a05aa0a82e91e3cab75506297e433e253d0"

So it really should be a new recipe, probably in meta-cmf-broadband if this is NOT BPi specific.